### PR TITLE
  Fix build of Logger.cpp with an unpatched version of gcc on Windows.  

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: src/libtomahawk/Config.h.in [hunk 1]
```diff
@@ -15,16 +15,18 @@
 
 #cmakedefine WITH_CRASHREPORTER
 #cmakedefine WITH_BINARY_ATTICA
-#cmakedefine WITH_QtSparkle
 #cmakedefine WITH_UPOWER
 #cmakedefine WITH_GNOMESHORTCUTHANDLER
 
 #cmakedefine LIBLASTFM_FOUND
 #cmakedefine QCA2_FOUND
+#cmakedefine QtSparkle_FOUND
```
Comment (TheOneRing) [inline src/libtomahawk/Config.h.in]: Did I do it right?

Comment (dschmidt) [inline src/libtomahawk/Config.h.in]: No. WITH and FOUND is different. WITH can be manually switched off. Why did you change it?

Comment (TheOneRing) [inline src/libtomahawk/Config.h.in]: then cmake should fail (REQUIRED) when qtsparkle is not found. But it just fails during compile.

Comment (muesli) [inline src/libtomahawk/Config.h.in]: Actually, QtSparkle should be optional by design. Must fix the code that depends on it.

Comment (TheOneRing) [inline src/libtomahawk/Config.h.in]: the problem was that WITH_ is either the default value or the value defined by the user regardless whether the package was found.

Comment (dschmidt) [inline src/libtomahawk/Config.h.in]: Yeah, I was mistaken. (although I don't do mistakes usually of course :P)
